src/Vector.cpp

```diff
@@ -108,7 +109,7 @@ void Vector::incrementValues(int cell, int group_first, int group_last,
    * temporary array using mutual exclusion locks */
   omp_set_lock(&_cell_locks[cell]);
 
-#pragma omp simd
+#pragma omp simd aligned(vals)
```

Comment:
It is hard to guarantee vals memory aligned all the time since Vector::incrementValues is an auxiliary function and it might be called randomly. So try to give some warning if it is not aligned.

Reply:
It's only called once in the entire code for now, for the CMFD current tallying.
A warning there would blow up the log file to TBs immediately if it got triggered